website/README.md

```diff
@@ -1,6 +1,6 @@
 # Cloud Operations Sandbox Website
 
-The Stackdriver Sandbox website is currently deployed at [cloud-ops-sandbox.dev](https://cloud-ops-sandbox.dev).
+The Cloud Operations Sandbox website is currently deployed at [cloud-ops-sandbox.dev](https://cloud-ops-sandbox.dev) (formerly "stackdriver-sandbox.dev").
```

Comment:
I'm not sure if the formerly note needs to be here. If it is, I would expect it to be a hyperlink